mk/make_dpdk.sh

```diff
@@ -81,6 +81,7 @@ edit_dpdk_config CONFIG_RTE_LIBRTE_PMD_PCAP=n $NEWCONFIG
 edit_dpdk_config CONFIG_RTE_APP_TEST=n $NEWCONFIG
 edit_dpdk_config CONFIG_RTE_TEST_PMD=n $NEWCONFIG
 edit_dpdk_config CONFIG_RTE_IXGBE_INC_VECTOR=n $NEWCONFIG
+eidt_dpdk_config CONFIG_RTE_SCHED_COLLECT_STATS=y $NEWCONFIG
 
```

Comment:
By default in DPDK, RED-marked packet is not dropped so the following command seems necessary.

```
eidt_dpdk_config  CONFIG_RTE_SCHED_RED=y $NEWCONFIG
``